docs/config/README.md

```diff
@@ -129,6 +129,7 @@ The `aws` module shows the current AWS region and profile. This is based on
 | `style`           | `"bold yellow"` | The style for the module.                                                   |
 | `disabled`        | `false`         | Disables the `AWS` module.                                                  |
 | `displayed_items` | `all`           | Choose which item to display. Possible values: [`all`, `profile`, `region`] |
+| `region_aliases`  |                 | Table of region aliases to display in addition to the AWS name.             |
```

Comment:
I think the convention for the docs is to put the `style`  and `disabled` vars at the bottom of the table.